### DISCUSSION TITLE
Cannot preview in powershell

### DISCUSSION THREAD

**Discussion by brm202322** [thegetty/quire#966] [Get Help]
When I enter quire preview into Microsoft powershell, I get the error:
 Cannot find module 'C...@11tyy\eleventy\cmd.js'
at Module._resolveFilename (node:internal/modules/cjs/loader:1140:15)
    at Module._load (node:internal/modules/cjs/loader:981:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []

I verified that the version of Node is correct, redownloaded the module, and verified that the file is in the folder and the path is correct.

**Comment by geealbers** [comment]
Hi @brm202322 can you take a screenshot of the contents of the quire project folder you created, or run the command `ls` inside Powershell while you're in the project folder, and share the results here? We've had some Windows users report that new projects aren't being fully created when running `quire new` and I suspect that might be the case here too.

**Reply by brm202322** [reply]
<img width="485" alt="image" src="https://github.com/user-attachments/assets/147c1100-3b5e-4afb-ad2e-4442ad295998">

**Comment by Erin-Cecele** [comment]
@brm202322 you are unfortunately encountering a known bug for Windows users: https://github.com/thegetty/quire/discussions/962. But we do have a workaround! Please email me at quire@getty.edu and I can share next steps.

**Comment by Erin-Cecele** [comment]
Hi @brm202322 To help us better understand why so many Windows users are encountering the preview error, can you do me a favor and run the command `quire info --debug` and share the Powershell output with me? So far, we have been unable to replicate the error on our end.
